src/main/java/datawave/microservice/metadata/tablecache/TableCacheController.java

```diff
@@ -0,0 +1,78 @@
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.codahale.metrics.annotation.Timed;
+
+import datawave.core.common.result.AccumuloTableCacheStatus;
+import datawave.webservice.result.VoidResponse;
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+@Tag(name = "Table Cache Controller /v1", description = "DataWave Table Cache Operations",
+                externalDocs = @ExternalDocumentation(description = "Query Executor Service Documentation",
+                                url = "https://github.com/NationalSecurityAgency/datawave-query-executor-service"))
+@RestController
+@RequestMapping(path = "/v1/AccumuloTableCache", produces = MediaType.APPLICATION_JSON_VALUE)
+@ConditionalOnProperty(name = "datawave.table.cache.enabled", havingValue = "true", matchIfMissing = true)
+public class TableCacheController {
+    private final Logger log = Logger.getLogger(TableCacheController.class);
+    private final TableCacheReloadService service;
+    
+    public TableCacheController(TableCacheReloadService service) {
+        this.service = service;
+    }
+    
+    /**
+     * <strong>JBossAdministrator or Administrator credentials required.</strong> Force a reload the table cache for the specified table.
+     *
+     * @return a VoidResponse
+     * @RequestHeader X-ProxiedEntitiesChain use when proxying request for user
+     * @RequestHeader X-ProxiedIssuersChain required when using X-ProxiedEntitiesChain, specify one issuer DN per subject DN listed in X-ProxiedEntitiesChain
+     * @HTTP 200 success
+     * @HTTP 500 internal server error
+     */
+    @Operation(summary = "Force a reload of the table cache for the specified table.")
+    @Timed(name = "dw.table.cache.reloadCache", absolute = true)
+    @Secured({"AuthorizedUser", "AuthorizedQueryServer", "AuthorizedServer", "InternalUser", "Administrator", "JBossAdministrator"})
+    @RequestMapping(path = "/reload/{tableName}", method = {RequestMethod.GET},
+                    produces = {"application/xml", "text/xml", "application/json", "text/yaml", "text/x-yaml", "application/x-yaml", "text/html"})
+    public VoidResponse reloadCache(@PathVariable String tableName) {
+        log.debug("Handling reload request for " + tableName + " and notifying other executor services");
+        service.reloadTable(tableName, true);
+        VoidResponse response = new VoidResponse();
+        response.addMessage(tableName + " reloaded and message sent to all other executor services");
+        return response;
+    }
+    
+    /**
+     * <strong>JBossAdministrator or Administrator credentials required.</strong> Get the status of the table caches
```

Comment:
It seems that more roles than these are listed in the Secured tag.  Same for the reloadCache.

Reply:
interesting.  Looking at AccumuloTableCache on integration it has the same comment, but also has the same roles as I have in the Secured tag.  What do we think is the right thing to do here?

Reply:
for reference:
https://github.com/NationalSecurityAgency/datawave/blob/integration/web-services/common/src/main/java/datawave/webservice/common/cache/AccumuloTableCache.java

Reply:
doing a blame it looks like the original code has always been that way.  Unless anyone has a better suggestion I'll probably just delete that comment.

Reply:
removed that comment...